### PR TITLE
VC-4210 add accounts table

### PR DESCRIPTION
**Pull request by alex-bodnar**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by PavelVlasenkoGo** [comment]
LGTM

File: database/schema/11-accounts.sql [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+DROP TABLE IF EXISTS vipcoin_chain_accounts_accounts CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_set_kinds CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_set_affiliate_address CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_register_user CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_account_migrate CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_set_affiliate_extra CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_set_extra CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_set_state CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_add_affiliate CASCADE;
+DROP TABLE IF EXISTS vipcoin_chain_accounts_create_account CASCADE;
```
Comment (IamStubborN) [inline database/schema/11-accounts.sql:102]: в bdjuno не используется мигратор, down миграции не нужны

File: dbconfig.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+development:
+  dialect: postgres
+  datasource: host=localhost port=5432 user=postgres dbname=juno password=postgres sslmode=disable
+  dir: database/schema
+
+dev:
+  dialect: postgres
+  datasource: host=10.10.1.79 port=5432 user=postgres dbname=juno password=postgres sslmode=disable
+  dir: database/schema
```
Comment (IamStubborN) [inline dbconfig.yml:9]: ну конфиг тоже

**Review by IamStubborN — CHANGES_REQUESTED** [review]

File: database/schema/11-accounts.sql (hunk 1 above)
Comment (alex-bodnar) [reply database/schema/11-accounts.sql:102]: это для sql-migrate, через него удобно миграции прогонять

File: dbconfig.yml (hunk 2 above)
Comment (alex-bodnar) [reply dbconfig.yml:9]: это для sql-migrate, через него удобно миграции прогонять

Comment (IamStubborN) [reply dbconfig.yml:9]: да, понимаю, но мы будем скоупом эти миграции вливать

либо нужно полностью прикручивать в коде возможность работы через sql-migrate

Comment (IamStubborN) [reply dbconfig.yml:9]: давай оставим, после разработки подумаем как лучше

**Comment by IamStubborN** [comment]
LGTM

**Review by IamStubborN — APPROVED** [review]

**Pull request merged**